assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -492,6 +492,10 @@ che.workspace.plugin_registry_url=NULL
 # plugins dependencies to a workspace
 che.workspace.plugin_broker.image=eclipse/che-plugin-broker:latest
 
+# Workspace tooling plugins registry endpoint. Should be a valid HTTP URL that includes protocol, port etc.
```

Comment:
I think `that includes protocol, port etc.` doesn't make property format clear enough. I would be useful to have example value as well.